cuda_pathfinder/mypy.ini

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+[mypy]
```

Comment:
nitpick: can we move this to a `[tool.mypy]` section in pyproject.toml instead of having a separate ini file here?

Reply:
Done: commit b73f42133a9b023073b0a80d6f433361c8f457a5

(that's a "today I learned")